src/cache/InvalidationPolicyCache.ts

```diff
@@ -546,15 +548,17 @@ export default class InvalidationPolicyCache extends InMemoryCache {
   // a list of the dereferenced matching entities from the cache based on the given fragment.
   readFragmentWhere<FragmentType, TVariables = any>(options: Cache.ReadFragmentOptions<FragmentType, TVariables> & {
     filter?: FragmentWhereFilter<FragmentType>;
+    limit?: number;
   }): FragmentType[] {
-    const { fragment, filter, ...restOptions } = options;
+    const { fragment, filter, limit, ...restOptions } = options;
```

Comment:
Maybe worth considering also adding an `offset` property; doesn't seem like much additional lift and would effectively allow for paging.